setup.cfg

```diff
@@ -15,14 +15,14 @@ classifiers =
 
 [options]
 packages = find:
-include_package_data = true
```

Comment:
It's an intentional feature of my packages that they include their data, even though this one doesn't. Since it seems that excluding the package data is required to exclude the package, that's highly undesirable... and would create technical debt (precluding adding data in the future and adding the surprise that it's not included implicitly as it is with other packages). That makes me sad.